### PR TITLE
RELATED: RAIL-4003 Add note on AttributeFilterButton limitations

### PR DESCRIPTION
**Pull request by plavemar**
JIRA: RAIL-4003, RAIL-4005

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/987/ : SUCCESS in 1m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/656/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/988/ : SUCCESS in 1m 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/657/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/990/ : SUCCESS in 1m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/659/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/992/ : SUCCESS in 1m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/661/ : SUCCESS in 6s

File: docs/10_vis__attribute_filter_button_component.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ sidebar_label: Attribute Filter Button
 copyright: (C) 2021 GoodData Corporation
 ---
 
+> **Limitations**:
```
Comment (evgeniabogranskaya) [inline docs/10_vis__attribute_filter_button_component.md:8]: I suggest two things:

1. Update the text to read this:

> When implementing the Attribute Filter Button component, consider the following:
> - The GoodData platform supports only filters with attribute elements defined by their URIs.
> - GoodData.CN supports filters with attribute elements defined by their `primary key` that is equal to the title of the respective element.

2. Put this text after the sentence "Optionally, you can define what attribute values should be selected in the filter by default.", before the example section.

**Comment by evgeniabogranskaya** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/993/ : SUCCESS in 1m 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/662/ : SUCCESS in 6s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No issue found. Congratulations!

More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR496

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/994/ : SUCCESS in 1m 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/200/ : SUCCESS in 46s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/95/ : SUCCESS in 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/290/ : SUCCESS in 37s

**Pull request merged**